blocks/sidedrawer/sidedrawer.js

```diff
@@ -0,0 +1,36 @@
+import { div, button, domEl } from '../../scripts/dom-helpers.js';
+
+export default function decorate(block) {
+  block.classList.add('calcite-mode-dark');
```

Comment:
Is this needed? Is this not supposed to use the class inherited from the body?

Reply:
You are right. Updated